### PR TITLE
default-util: Fix log dir errors.

### PR DESCRIPTION
**Pull request by priyank-p**

### CODE REVIEW

**Comment by akashnimare** [comment]
I don't think it has to do with main vs remote app process. Getting the same error.

File: app/renderer/js/utils/domain-util.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const JsonDB = require('node-json-db');
 const request = require('request');
 const Console = require('../console');
 
-const console = new Console({
+const logger = new Console({
 	file: `domain-util.log`
```
Comment (akashnimare) [inline app/renderer/js/utils/domain-util.js]: @cPhost let's add the timestamp here.

File: app/renderer/js/utils/default-util.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+	app = require('electron').app;
+}
 
+const zulipDir = app.getPath('userData');
+const logDir = `${zulipDir}/Logs/`;
 const initSetUp = () => {
+	// if it is the first time the app is running
+	// create zulip dir in userData folder to
+	// avoid errors
+	if (!fs.existsSync(zulipDir)) {
```
Comment (akashnimare) [inline app/renderer/js/utils/default-util.js]: The one thing that worries me here is that the app directory is ideally created by the electron. I hope this doesn't get messed up with that thing 😕

Comment (priyank-p) [reply app/renderer/js/utils/default-util.js]: I confirmed that it will add all the files are added afterwards.

Comment (akashnimare) [reply app/renderer/js/utils/default-util.js]: @cPhost so we don't need to worry, right?

Comment (priyank-p) [reply app/renderer/js/utils/default-util.js]: Yep, to be 100% sure try an packaged app.

**Comment by akashnimare** [comment]
Can you squash the commits here so that I can merge it?

**Comment by akashnimare** [comment]
Merged, thanks @cPhost 🚀

**Pull request merged**